### PR TITLE
ie8_server2003 verb request

### PR DESCRIPTION
**Pull request by PietJankbal**
ie8_server2003 verb request as pull request,

### CODE REVIEW

File: src/winetricks [hunk 1]
```diff
@@ -10142,6 +10142,139 @@ load_ie8_kb2936068()
 
 #----------------------------------------------------------------
 
+w_metadata ie8_server2003 dlls \
+    title="Internet Explorer 8 WindowsServer2003" \
+    publisher="Microsoft" \
+    year="?" \
```
Comment (Kreyren) [inline src/winetricks]: Missing year metadata

**Comment by PietJankbal** [comment]
whitespace errors in this one

**Comment by PietJankbal** [comment]
whitespace errors in this one

**Comment by Kreyren** [comment]
Replaced by https://github.com/Winetricks/winetricks/pull/1437

**Pull request closed without merging**